File: java/lib/src/main/java/com/svix/ApplicationListOptions.java
package com.xwebhook;

import com.xwebhook.models.Ordering;

public class ApplicationListOptions extends ListOptions {
    private Ordering order;

    public void setOrder(final Ordering order) {
        this.order = order;
    }

    public Ordering getOrder() {
        return this.order;
    }
}

File: kotlin/lib/src/main/kotlin/ApplicationListOptions.kt
package com.xwebhook.kotlin

import com.xwebhook.kotlin.models.Ordering

class ApplicationListOptions : ListOptions() {
    var order: Ordering? = null

    fun order(order: Ordering) = apply { this.order = order }

    override fun iterator(iterator: String): ApplicationListOptions = apply { super.iterator(iterator) }

    override fun limit(limit: Int) = apply { super.limit(limit) }
}
